t/04-openresty/ffi/200-proxy_wasm_and_lua_sanity.t

```diff
@@ -442,6 +442,7 @@ qr/.*? wasm cleaning up request pool.*
 
 
 === TEST 9: proxy_wasm FFI - filter chain attached before and during error handler location
+--- SKIP
```

Comment:
Still a TODO: since `next_id` is not global anymore, this can't work anymore either.